README.md

```diff
@@ -516,6 +517,25 @@ as a `debug-each` script at the project, backend, and suite levels, so they
 are aggregated and repeated for every task under them.
 
 
+<a name="reexecuting"/>
+
+## Reexecuting
+
+Reproduce an error could be a very boring experience, and Spread has a way to
+simplify that process by reexecuting the tasks as many times is desired until
+the task fails.
+
+To do that there are two options `-retries`, and `-stop-on-fail` which are
+used to determine the number of time to reexecute a task and if those executions
+have to be stoped when the task has failed.
+
+The `-retries` option receives an integer which indicates the number of retries 
+to do, being 0 the default value, and -1 can be used to reexecute indefinitely.
```

Comment:
Let's please drop the -1 special behavior and that paragraph. People can easily type 100 or 10000.